pkg/expapi/types.go

```diff
@@ -361,3 +361,34 @@ type ThirdPartyResourceDataList struct {
 	// Items is a list of third party objects
 	Items []ThirdPartyResourceData `json:"items"`
 }
+
+// LimitCapability defines the privilege and capability limits of PodSpecs in a namespace
```

Comment:
we already have the concept of users in the authenticator/authorizer interfaces... see main thread for discussion about namespace+admission vs user+policy application of this, and the intersections of those cases